### ISSUE TITLE
Text hint in the search entry is hardly visible 

### ISSUE THREAD

**Issue by mivoligo** [ubuntu/codelabs#8]
The text hint in the search area is grey on orange background. It does not look good and is a bit hard to see. Screenshot attached.
![screenshot at 15-30-56](https://cloud.githubusercontent.com/assets/1629120/20488709/fa3b75a8-afff-11e6-8c5b-77800920be57.png)

**Comment by didrocks** [comment]
Agreed :) There is some design work on the codelab itself to get the whole UI changed. This is a first beta experiment and that issue will get fixed :)
